engine/lib/phx/src/engine/event_bus.rs

```diff
@@ -40,18 +40,22 @@ impl Default for UpdatePass {
     }
 }
 
-#[luajit_ffi_gen::luajit_ffi(repr = "u16")]
+#[luajit_ffi_gen::luajit_ffi(repr = "u8")]
```

Comment:
I think `repr` is not necessary - proc macro deduct it itself

Reply:
roger